src/frontends/paddle/src/op/sum.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs sum(const NodeContext& node) {
+    auto data = node.get_ng_inputs("X");
+    const auto input_type = data[0].get_element_type();
+    const auto shape = data[0].get_shape();
```

Comment:
I don't see that you use `shape` and `input_type` here. I think we should remove these variables

Reply:
Done, already removed these redundant codes.